docs/en/Running-Presto-with-Alluxio.md

```diff
@@ -9,52 +9,95 @@ priority: 2
+#### Configure Presto to connect to Hive Metastore
 
-Please [Download Presto](https://repo1.maven.org/maven2/com/facebook/presto/presto-server/)(This doc uses presto-0.191). Also, please complete Hive setup using
-[Hive On Alluxio](Running-Hive-with-Alluxio.html)
+Presto gets the database and table metadata information, as well as
+the file system location of table data from Hive Metastore.
+Edit the Presto configuration `${PRESTO_HOME}/etc/catalog/hive.properties`:
 
-## Configuration
+```properties
+connector.name=hive-hadoop2
+hive.metastore.uri=thrift://localhost:9083
+```
 
-Presto gets the database and table metadata information from Hive Metastore. At the same time,
-the file system location of table data is obtained from the table's metadata entries. So you need to configure
-[Presto on HDFS](https://prestodb.io/docs/current/installation/deployment.html). In order to access HDFS,
-you need to add the Hadoop conf files (core-site.xml,hdfs-site.xml), and use `hive.config.resources` in
-file `/<PATH_TO_PRESTO>/etc/catalog/hive.properties` to point to the file's location for every Presto worker.
+#### Distribute the Alluxio client jar to all Presto servers
 
-### Configure `core-site.xml`
+Put Alluxio client jar `{{site.ALLUXIO_CLIENT_JAR_PATH}}` into directory
+`${PRESTO_HOME}/plugin/hive-hadoop2/`
+(this directory may differ across versions) on all Presto servers. Restart the process of
+coordinator and worker.
 
-You need to add the following configuration items to the `core-site.xml` configured in `hive.properties`:
+### Advanced Setup
+
+#### Configure additional Alluxio properties
+
+To configure additional Alluxio properties, one can add them to the Hadoop conf files 
+(`core-site.xml`, `hdfs-site.xml`), and use 
+Presto property `hive.config.resources` in
+file `${PRESTO_HOME}/etc/catalog/hive.properties` to point to the file's location for every Presto
+worker.
+
+```
+hive.config.resources=/<PATH_TO_CONF>/core-site.xml, /<PATH_TO_CONF>/hdfs-site.xml
+```
+
+For example, one can add additional Alluxio properties to `core-site.xml` to 
+change `alluxio.user.file.writetype.default` from default `MUST_CACHE` to `CACHE_THROUGH`:
 
 ```xml
 <property>
-  <name>fs.alluxio.impl</name>
-  <value>alluxio.hadoop.FileSystem</value>
-</property>
-<property>
-  <name>fs.AbstractFileSystem.alluxio.impl</name>
-  <value>alluxio.hadoop.AlluxioFileSystem</value>
-  <description>The Alluxio AbstractFileSystem (Hadoop 2.x)</description>
+  <name>alluxio.user.file.writetype.default</name>
+  <value>CACHE_THROUGH</value>
 </property>
 ```
 
-To use fault tolerant mode, set the Alluxio cluster properties appropriately in an
+Alternatively, you can also append the conf path (i.e. `${ALLUXIO_HOME}/conf`) containing [`alluxio-site.properties`](Configuration-Settings.html) to Presto's JVM config at `etc/jvm.config` under Presto folder. The advantage of this approach is to have all the Alluxio properties set within the same file of `alluxio-site.properties`.
```

Comment:
Can we recommend one approach for setting properties? Otherwise users can be confused about which to take. The recommended approach should be described first so that users don't need to read about other approaches unless the recommended approach doesn't work for them.

Reply:
Agree but unfortunately, I don't have a recommendation at this point. We can come back to give the recommendation later.